tests/main/views/test_marketplace.py

```diff
@@ -326,7 +325,7 @@ def test_cookie_page(self):
         assert res.status_code == 200
         assert '<h1>Cookies</h1>' in self._strip_whitespace(res.get_data(as_text=True))
 
-    def test_cookie_page(self):
+    def test_ts_n_cs_page(self):
```

Comment:
No harm in spelling this out long?  Took me a while to figure what on earth `ts_n_cs` meant.